src/reporter/github.js

```diff
@@ -141,23 +194,22 @@ export default class GitHub {
       }
 
       const managedLabelsNames = this.MANAGED_LABELS.map(label => label.name);
+
       const labelsNotManagedToKeep = issue.labels.map(label => label.name).filter(label => !managedLabelsNames.includes(label));
       const [managedLabel] = issue.labels.filter(label => managedLabelsNames.includes(label.name)); // It is assumed that only one specific reason for failure is possible at a time, making managed labels mutually exclusive
 
       if (issue.state !== GitHub.ISSUE_STATE_CLOSED && managedLabel?.name === label) {
         return;
       }
 
-      await this.updateIssue(issue, {
-        state: GitHub.ISSUE_STATE_OPEN,
-        labels: [ label, ...labelsNotManagedToKeep ],
-      });
-      logger.info(`Updated issue #${issue.number}: ${issue.html_url}`);
+      const updatedIssue = await this.updateIssue(issue, { state: GitHub.ISSUE_STATE_OPEN, labels: [ label, ...labelsNotManagedToKeep ].filter(label => label) });
+
       await this.addCommentToIssue({ issue, comment: description });
 
-      logger.info(`Added comment to issue #${issue.number}: ${issue.html_url}`);
+      this.issuesCache.set(updatedIssue.title, updatedIssue);
```

Comment:
I feel that the cache update should take place in `updateIssue`, not in the caller.

Reply:
Yes, of course. Fatigue...